nilearn/plotting/html_surface.py

```diff
@@ -126,6 +140,58 @@ def one_mesh_info(surf_map, surf_mesh, threshold=None, cmap=cm.cold_hot,
     return info
 
 
+def one_mesh_info_niivue(surf_map, surf_mesh, threshold=None, bg_map=None):
+    """Build dict for plotting one surface map on a single mesh."""
+    info = {}
+
+    # Create temporary directory
+    output_path = Path(tempfile.mkdtemp())
```

Comment:
I don't see this getting cleaned up?

Reply:
You're right! This was addressed in our latest commits.